compiler/util.py

```diff
@@ -9,6 +9,11 @@
 def flatten_list(lst):
     return [item for sublist in lst for item in sublist]
 
+def unzip(lst):
```

Comment:
`list(zip(*lst))`? Sorry for the code golf. (Will be degenerate on empty lists, returning `[]`.)

Reply:
I tried that first, but the problem is that it returnes a list of tuples, and not lists. This in turn led to some error somewhere (that I am sure can be fixed but I didn't want to spend the time on it) :)